CPAC/registration/registration.py

```diff
@@ -1177,8 +1177,8 @@ def create_wf_calculate_ants_warp(
                                         imports=reg_imports),
                 name='calc_ants_warp',
                 mem_gb=2.8,
-                mem_x=(1708448960473801 / 604462909807314587353088,
-                       'moving_brain'))
+                mem_x=(954925290115543 / 4722366482869645213696,
```

Comment:
Are these the :sparkles: magic numbers :sparkles: that were learned from your profiling of tasks? Is there a reason we don't the result of this ratio? E.g., `2e-7`?

Reply:
Nothing magic, just the slope from a linear regression :smile: :mage:

I opted for fractions since x*y*z*t can be pretty big. For https://github.com/FCP-INDI/C-PAC/blob/80424468c7f4e59c102f446b05d4154ec1cd4b2d/CPAC/pipeline/nipype_pipeline_engine/engine.py#L24

changing `mem_x` from the current fraction to 2e-7 would adjust the estimate from 20.78 to 20.58 for an unknown image (e.g., an image from S3 that hasn't been accessed yet)

(Both of those numbers seem high to me, but I strongly prefer stability to efficiency, and once https://github.com/FCP-INDI/C-PAC/issues/1688 is resolved, these estimates will only be relevant for runs without a `--runtime_usage`)

Reply:
Sounds great; thanks, Jon!